contract/src/main/kotlin/io/provenance/scope/loan/contracts/RecordLoanContract.kt

```diff
@@ -77,6 +77,9 @@ open class RecordLoanContract(
                                 newLoan.originatorName.isNotBlank() orError "Loan is missing originator name",
                             )
                             uliValidation(newLoan.uli)
+                            newLoan.funding.takeIf { it.isSet() }?.let { newLoanFunding ->
+                                fundingValidation(newLoanFunding)
+                            }
```

Comment:
Is the funding field expected to be set whenever the loan is recorded using `RecordLoanContract`? If so, this should be replaced with
```suggestion
                            fundingValidation(newLoanFunding)
```

Reply:
I'm going to lean toward allowing it to not be set in case there's a use case involving an unfunded loan being boarded before being passed off to another service for funding.